packages/cursorless-engine/src/actions/BringMoveSwap.ts

```diff
@@ -34,7 +39,7 @@ abstract class BringMoveSwap {
   protected abstract decoration: {
     sourceStyle: FlashStyle;
     destinationStyle: FlashStyle;
-    getSourceRangeCallback: (target: Target) => Range;
+    getSourceRangeCallback?: (target: Target) => GeneralizedRange;
```

Comment:
Made this optional. If you don't pass a callback into flashTargets() the content range is used by default.